tests/config/templates/condition.yaml

```diff
@@ -0,0 +1,86 @@
+{{- define "condition" }}
+  {{- $root := index . 0 }}
+  {{- $condition := index . 1 }}
+  {{- $expect := index . 2 }}
+  {{- if eq (include "_config.condition" (list $root $condition)) $expect }}
+    {{- printf "# Condition %q is %q" $condition $expect }}
+  {{- else }}
+    {{- include "fail" (list $root (printf "Condition %q is NOT %q" $condition $expect)) }}
```

Comment:
Here we probably want to also print the _actual_ value…

Reply:
Fixed in d120de6